app/hotels/src/map/allHotels/MapView.js

```diff
@@ -46,18 +46,9 @@ export class Map extends React.Component<Props, State> {
     };
   }
 
-  componentDidMount() {
-    const { selectedIndex } = this.props;
-    // it's necessary to call showCallout to bring selected marker to foreground
-    if (this.markers[selectedIndex]) {
-      this.markers[selectedIndex].showCallout();
-    }
-  }
-
-  componentDidUpdate() {
-    this.animateToCoordinate(this.props.selectedIndex);
-    if (this.markers[this.props.selectedIndex]) {
-      this.markers[this.props.selectedIndex].showCallout();
+  componentDidUpdate(prevProps: Props) {
+    if (prevProps.selectedIndex !== this.props.selectedIndex) {
+      this.animateToCoordinate(this.props.selectedIndex);
```

Comment:
Are you sure this is not necessary anymore?

Reply:
I cant see any difference with or without it. Marker is not brought forward anyway.

Reply:
Ah okay :D Then I'm fine with it!